packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -84,16 +89,21 @@ tr > td:first-child {
   font-family: $font-sans;
 }
 
-.header-text div {
+.text-large div {
   @extend .text-xl;
   @extend .font-sans;
   text-align: center !important;
-  @extend .mb-5;
```

Comment:
We're actually not extending any of these margin classes anywhere except for `mb-5`. I think by keeping it as a separate class, we don't have to think about overriding styles anymore and can just add a separate class wherever we'd like

Reply:
Hmm it might be a tad confusing to call this `text-xlarge` when we've got the `text-xl` class too.

I think we may want to avoid manually adding `mb` classes along with another class because it makes it easier to use the wrong spacing class which leads to inconsistency.  I don't mind a `text-header` class (that might be better than `header-text`) with your `.text-xlarge` styles + `mb-5` extended and then, use that class on the header at the top of every email, but I think that's basically what I recommended with the placeholder suggestion yesterday, heh.

BTW looks like this will happen anyway, but I'd say wait until your other PRs are merged so you can update these classes in those templates as well.

Reply:
Maybe we can have the classes `text-header`, `text-body`, `text-body-mb-5`, and `text-footer`. I like all of these except the third one but am not sure what else to call it right now. 😆

Reply:
Hmmm, right! But when I was working on converting `verify` email template, there was one more text section apart from header and body text and had styles essentially similar to the (new) `text-body` class but the font-color is darkish grey. See below

![image](https://user-images.githubusercontent.com/37609519/130111789-d13b7d5f-7be0-4584-8f6c-4ab71fad054e.png)

What do you think we should do in such situations? Not related to this PR ofc but we can potentially avoid refactoring the styles again in future PRs

Reply:
Huh, we could call it `text-body-grey` perhaps? Does it match the exact styles as `text-footer`?

Reply:
I think the styles are similar to body text but yeah `text-body-grey` sounds good to me.